composer.json

```diff
@@ -37,7 +35,9 @@
         "zendframework/zend-session": "^2.6.2",
         "zendframework/zend-mvc-plugin-flashmessenger": "^0.1.0",
         "zendframework/zend-mvc-console": "^1.1.8",
-        "zendframework/zend-validator": "^2.8"
+        "zendframework/zend-validator": "^2.8",
+        "zendframework/zend-coding-standard": "~1.0.0",
+        "phpunit/phpunit": "^4.6 || ^5.0"
```

Comment:
phpunit should in require path to manage syntax compat, also, use phpunit ^5.6 || ^6.0 in favor of https://github.com/zendframework/zend-test/pull/43